App.js

```diff
@@ -1,26 +1,32 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React, { useState, useEffect } from 'react';
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
-
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
+import Question from './components/Question';
+import Answer from './components/Answer';
+import { ShakeEventExpo } from './components/ShakeEventExpo';
 
 const App = () => {
+  const [firstView, setFirstView] = useState(true);
+
+  const shakePhone = () => {
+    setFirstView(false);
+  };
+
+  useEffect(() => {
+    ShakeEventExpo.addListener(() => {
+      shakePhone();
+    });
+
+    return () => {
+      ShakeEventExpo.removeListener();
```

Comment:
Great job, it's very important to perform listeners cleanup after we unmount a component :)